community/bolt/src/main/java/org/neo4j/bolt/v1/messaging/PackStreamMessageFormatV1.java

```diff
@@ -136,6 +136,20 @@ public void handleRecordMessage( Record item )
                 packer.pack( field );
             }
             onMessageComplete.onMessageComplete();
+
+            //The record might contain unpackable values,
+            //hence we must consume any errors that might
+            //have occurred.
+            packer.consumeError( ( status, s ) -> {
+                try
+                {
+                    handleFailureMessage( status, s );
```

Comment:
This is a bit unsafe; the state machine won't find out that this message has gotten sent, and may send more messages after this. We can get around that by throwing an exception after we write the failure message, like BoltIOException, something that'll trigger the caller to see this case as a fatal IO error and close the channel and associated session stuff. Eg; the end result ends up being:

    > RUN "CALL getInvalidNode"
    < SUCCESS ..
    < RECORD Node(null)
    < FAILURE
    < [eof]